packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -1142,14 +1138,11 @@ describe("Integration tests", () => {
         await addOneAgreement(agreementEservice1, agreements);
 
         await tenantService.verifyVerifiedAttribute({
-          tenantId: tenantWithVerifiedAttribute.id,
+          tenantId: targetTenant.id,
```

Comment:
Why this change? I find more correct to use `tenantWithVerifiedAttribute` since it's the tenant that is present in the readModel